src/framework/util/index.ts

```diff
@@ -8,6 +8,14 @@ export function now(): number {
   return perf.now();
 }
 
+export function rejectOnTimeout(ms: number, msg: string): Promise<GPUUncapturedErrorEvent> {
```

Comment:
This seems weird to have GPUUncapturedErrorEvent in until don't you think?

Reply:
Oops, yes.